S3Scanner/S3Service.py

```diff
@@ -49,7 +49,9 @@ def __init__(self, forceNoCreds=False, endpoint_url='https://s3.amazonaws.com',
                 raise InvalidEndpointException(message=f"Endpoint '{self.endpoint_url}' does not appear to be S3-compliant")
 
         # Check for AWS credentials
-        session = botocore.session.get_session()
+        session = boto_session.Session()
+        if profile in session.available_profiles: # use provided profile, if it is availble to use
+            session = boto_session.Session(profile_name=profile)

```

Comment:
Add an `else` block after this. If the user specified a profile name that isn't available, print an error and exit:
`Error: profile "(profile_name_here)" not found in ~/.aws/credentials`

Reply:
@sa7mon , I went ahead and added a commit that takes care of this, good call!